modules/nodes-cma-autoscaling-custom-prometheus-config.adoc

```diff
@@ -48,7 +49,26 @@ where:
 +
 <service_account>:: Specifies the name of the service account.
 
-. Use the following command to locate the token assigned to the service account:
+. Use the following command to create the service token for the service account:
++
+[source,terminal]
+----
+$ oc apply -f - <<EOF
+apiVersion: v1
+kind: Secret
+metadata:
+  name: thanos-token
+  annotations:
+    kubernetes.io/service-account.name: <service_account>
+  type: kubernetes.io/service-account-token
+EOF
+----
++
+where:
++
+<service_account>:: Specifies the name of the service account.
```

Comment:
Sorry, I know this merged, but didn't a chance to review.

Might want to consider a future fix for the same comments I had in: https://github.com/openshift/openshift-docs/pull/78003#pullrequestreview-2136877698

The `<service_account>` is fine (I suggested to change it to `thanos` because we reference it as `thanos` later...), that's just bit of a nit pick. However, down below, there is a minor inconsistency.

When you create the `Secret` with `name: thanos-token` to create the service token, `thanos-token` will be the token name. I think it may be slightly confusing to users that the docs still have a generate name, which is not the right name. I'd think most users would realize this, but I think it's better to be precise to avoid confusion.

[Line 93](https://github.com/openshift/openshift-docs/pull/78467/files#diff-5b0981a1b66b58b9f7f05e5c821abb40dd2c10a4c2d48e83ddf5e09db34224acR93):
```
Tokens: thanos-token-9g4n5 <1>
```
Should be:
```
Tokens: thanos-token <1>
```

And [line 112](https://github.com/openshift/openshift-docs/pull/78467/files#diff-5b0981a1b66b58b9f7f05e5c821abb40dd2c10a4c2d48e83ddf5e09db34224acR112) for the TriggeredAuthentication creation:
```
    name: thanos-token-9g4n5 <3>
```
Should be:
```
    name: thanos-token <3>
```

Reply:
no worries, appeciate it Grant! i'm in touch with @mburke5678 and it looks like he's implemented those changes into his PR on top of the new ones 😄

Reply:
Oh I see now. Sounds good, thanks!